docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+However, depending on your choice, there's already some examples in the **Epinio Howtos** section which contains links to the respective clusters' installation docs.
+
+:::
+
+In this tutorial, we'll use [Rancher Desktop](../howtos/install_epinio_on_rancher_desktop.md) as our local Kubernetes cluster. However you should be able to follow this tutorial with the local Kubernetes cluster of your choice.
+
+If not already done, you can install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) of Rancher Desktop for your operating system.
+
+### [Optional] Additional binaries
+
+Two additional binaries need to be installed in your system:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) for communicating with the Kubernetes cluster
+
+- [helm](https://helm.sh/docs/intro/install/) for deploying the Epinio Helm Charts
+
+Depending on the local Kubernetes cluster you installed (i.e. Rancher Desktop), these two binaries might be already installed.
+
+## Installation
+
+Once you have your local Kubernetes cluster installed and running, you can [install Epinio](../installation/install_epinio.md).
+
+Here are the steps for Rancher Desktop:
+
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+
+# Wait for cert-manager to stabilize
+
+helm repo add epinio https://epinio.github.io/helm-charts
+
+helm repo update
+
+helm install epinio -n epinio --create-namespace epinio/epinio --set global.domain=127.0.0.1.sslip.io
+```
+
+### CLI
+
+To interact with your Epinio installation, you've to download the [Epinio CLI binary](https://github.com/epinio/epinio/releases/latest). The binary is available for the three main Operating Systems (OS), so pick the one suited for your own OS.
+
+### Login
+
+The first task to perform after Epinio installation, is to [login](../references/commands/cli/epinio_login.md) with the binary you downloaded:
+
+```shell
+epinio login -u admin 'https://epinio.127.0.0.1.sslip.io'
+
+# Trust the certificate by pressing 'y' and 'enter'
+```
+
+:::tip
+
+If your local Kubernetes cluster restarts, you only need to login again with the command above. Epinio stays installed and the certificates are still valid.
+
+:::
+
+You can confirm that you're logged in by checking the Epinio settings:
+
+```shell
+epinio settings show
+```
+
+Alternatively, you can also open the Epinio URL, <https://epinio.127.0.0.1.sslip.io>, in your preferred browser and use the web UI.
+
+## Deploy your application with Epinio
+
+Now that the "operational" tasks are done, it's time to concentrate on the most important task: use Epinio to deploy your application.
+
+As [explained here](../explanations/detailed-push-process.md#7-stage), Epinio uses [Paketo buildpacks](https://paketo.io/) to create a container image for your application. This image is then used to create a container with your application, which will run on your local Kubernetes cluster.
```

Comment:
I'd start with "Epinio uses Paketo..." and pull the "as explained here" to the end of the line as "more details about this can be found here"

Reply:
I.e., likely
```suggestion
Epinio uses [Paketo buildpacks](https://paketo.io/) to create a container image for your application. This image is then used to create a container with your application, which will run on your local Kubernetes cluster. More about this can be found in the explanations of the [detailed push process](../explanations/detailed-push-process.md#7-stage).
```